cognite/neat/_session/_prepare.py

```diff
@@ -38,22 +38,68 @@ def cdf_compliant_external_ids(self) -> None:
                 )
             )
 
-    def to_extension(
+    def to_enterprise(
         self,
-        new_data_model_id: DataModelIdentifier,
-        org_name: str | None = None,
-        mode: Literal["extension", "solution"] = "extension",
+        data_model_id: DataModelIdentifier,
+        org_name: str = "My",
+        dummy_property: str = "GUID",
     ) -> None:
-        """Uses the current data model as a basis to extend from.
+        """Uses the current data model as a basis to create enterprise data model
 
         Args:
-            new_data_model_id: The new data model that is extending the current data model.
-            org_name: Organization name to use for the views in the new data model. This is required if you are
-                creating an extension from a Cognite Data Model.
+            data_model_id: The enterprise data model id that is being created
+            org_name: Organization name to use for the views in the enterprise data model.
+
+        !!! note "Enterprise Data Model Creation"
+            Always create an enterprise data model from a Cognite Data Model as this will
+            assure all the Cognite Data Fusion applications to run smoothly, such as
+            - Search
+            - Atlas AI
+            - ...
+
+        """
+        if dms := self._state.last_verified_dms_rules:
```

Comment:
I am a bit skeptical about this pattern we started to use as it will silently do nothing if there is no verified last dms rules.